video/decode/vd_lavc.c

```diff
@@ -156,14 +158,6 @@ static const struct vd_lavc_hwdec mp_vd_lavc_rpi_copy = {
 };
 #endif
 
-#if HAVE_ANDROID
-static const struct vd_lavc_hwdec mp_vd_lavc_mediacodec = {
```

Comment:
wouldn't it make sense to move/keep the `vd_lavc_hwdec mp_vd_lavc_mediacodec_copy` definition here?